snc.sh

```diff
@@ -290,7 +291,10 @@ fi
 ${UNZIP} -o -d openshift-clients/windows/ openshift-clients/windows/oc.zip
 OC=./openshift-clients/linux/oc
 
-run_preflight_checks
+if [[ ${CERT_ROTATION} == "true" ]]
+then
+    run_preflight_checks
+fi
```

Comment:
Why disable preflight checks if cert rotation is disabled? I'd expect `renew_certificates` to be disabled?

Reply:
Good catch, it should be `renew_certificates` :(